internal/controller/contract.go

```diff
@@ -159,6 +159,7 @@ func (c *ContractController) watcherStart(ctx context.Context, client *ethclient
 			return
 		case <-c.stopL2ContractChan:
 			log.Info("ContractController l2 watch the run loop exit")
+			return
```

Comment:
Ok, but why do we need 2 stop channels in the first place?

Reply:
you are right. I'll merge them into one.

Reply:
use one channel to exit in [6bfc3b8](https://github.com/scroll-tech/chain-monitor/pull/122/commits/6bfc3b8c5e39c7463fabc3c66a1f27ba83314778).